docs/ref/dotq.md

```diff
@@ -1224,6 +1225,33 @@ it recursively as in [`load`](load.md), but into the default namespace.
 (Implements system command [`\l`](../basics/syscmds.md#l-load-file-or-directory).)
 
 
+## `lo` (load without)
+
+```syntax
+.Q.lo[`database;cd;scripts]
+```
+
+load a database without changing directory and/or loading scripts in the database (since 4.1t 2023.03.01)
```

Comment:
Should this be a capital?

```suggestion
Load a database without changing directory and/or loading scripts in the database (since 4.1t 2023.03.01)
```